docs/api-reference/traces.md

```
@@ -0,0 +1,225 @@
+| Status Code | Description |
+|-------------|-------------|
+| **200** | Successful Response |
+| **422** | Validation Error |
+
+**Response Body** (application/json):
+
+| Name | Type | Description |
+|------|------|-------------|
+| `id` | string (uuid) | - |
+| `name` | string | - |
+| `status` | string = `unset` \| `ok` \| `error` | OpenTelemetry status codes.
```

markdownlint-cli2 (0.22.0)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)